addons/stock_account/models/product.py

```diff
@@ -235,7 +236,7 @@ def _change_standard_price(self, new_price, counterpart_account_id=False):
         account_moves.post()
 
         # Actually update the standard price.
-        self.write({'standard_price': new_price})
+        self.with_context(force_company=company_id.id).sudo().write({'standard_price': new_price})
```

Comment:
force_company is probably useless since sudo ref

Reply:
i'm not sure. it's still used almost everywhere (find pricelist, in ir.property, _select_seller,..)